Document how to run dev env (#282)
src-d/sourced-ui: CONTRIBUTING.md

    The first time you launch it, it will take some time to build all the UI assets,
    you can see the progress of the build from `sourced-ui` logs (see next step).

>     It may happen that the hot reloading stop working with an error message

Reviewer: ```suggestion
    It may happen that the hot reloading stops working with an error message
```

>     `Failed to compile` that can not be solved modifying the source code.

Reviewer: ```suggestion
    `Failed to compile` that cannot be solved modifying the source code.
```

>     It use to happen when swiping between branches or stopping the watcher.

Reviewer: ```suggestion
    It usually happens when switching between branches or stopping the watcher.
```
